.circleci/config.yml

```diff
@@ -480,6 +498,50 @@ commands:
       - revenuecat/set-ssh-key-for-git:
           fingerprint: "SHA256:UYi3gsOInLRPZy6MPho7Yr/YK1VmjDvPGo7sJvIcnPM"
 
+  setup_xcconfig:
+    parameters:
+      api_key:
+        type: string
+        description: "The RevenueCat API key value"
+      force_server_error_strategy:
+        type: enum
+        enum: ["never", "primary_backend_down"]
+        default: "never"
+        description: "Server error strategy for testing fallback behavior"
+      api_host:
+        type: string
+        default: ""
+        description: "Optional API host for testing (e.g., 'api-staging.revenuecat.com')"
+      proxy_url_scheme:
+        type: string
+        default: ""
+        description: "Optional proxy URL scheme (e.g., 'https')"
+      proxy_url_host:
+        type: string
+        default: ""
+        description: "Optional proxy URL host (e.g., 'fortress-us-east-2.revenuecat.com')"
```

Comment:
It's great to have this parameter, but the example url given here for fortress 2 could be misleading I think, as in reality it's being passed as `api_host` and not as `proxy_url_host`. We could make up some proxy url for the description